src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,14 @@
 Release History
 ===============
 
+2.39.1
+++++++
+
+**CosmosDB**
```

Comment:
Don't modify history notes manually. This will be generated automatically in release pipeline.

To make sure the history notes correct, please format the title and PR description according to [submitting-pull-requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests)

Reply:
Corrected.